disk/development/ontology.ttl

```diff
@@ -302,6 +326,53 @@ xsd:date rdf:type rdfs:Datatype .
                     rdfs:comment "Property that links a line of inquiry to the hypothesis query text"@en ;
                     rdfs:label "has hypothesis query"@en .
 
+###  http://disk-project.org/ontology/disk#hasEmail
+:hasEmail rdf:type owl:DatatypeProperty ;
+                 rdfs:domain :Person ;
+                 rdfs:range xsd:string ;
+                 rdfs:comment "Email of a person"@en ;
+                 rdfs:label "has email"@en .
+
+###  http://disk-project.org/ontology/disk#hasName
+:hasName rdf:type owl:DatatypeProperty ;
+                 rdfs:domain :Person ;
+                 rdfs:range xsd:string ;
+                 rdfs:comment "Name of a person"@en ;
+                 rdfs:label "has name"@en .
+
+###  http://disk-project.org/ontology/disk#hasUsageNotes
+:hasUsageNotes rdf:type owl:DatatypeProperty ;
+             rdfs:domain [ rdf:type owl:Class ;
+                                      owl:unionOf ( :LineOfInquiry
+                                                    :Hypothesis
+                                                  )
+                                    ] ;
+             rdfs:range xsd:string ;
+             rdfs:comment "Property that add notes to Line of Inquiry and Hypothesis"@en ;
+             rdfs:label "has usage notes"@en .
+
+###  http://disk-project.org/ontology/disk#hasQuestionTemplate
+:hasQuestionTemplate rdf:type owl:DatatypeProperty ;
+               rdfs:domain :Question ;
+               rdfs:range xsd:string ;
+               rdfs:comment "property to store human readable template of hypothesis question"@en ;
```

Comment:
This is not grammatical. And is it human readable?
Property designed to store a readable template associated with a hypothesis question